node/src/main/scala/coop/rchain/node/NodeRuntime.scala

```diff
@@ -789,7 +796,7 @@ object NodeRuntime {
           conf.roundRobinDispatcher.giveUpAfterSkipped,
           conf.roundRobinDispatcher.dropPeerAfterRetries
         )
-      }
+      }*/
```

Comment:
Can we keep fairness constraint in code but disable it with CLI arguments?

Reply:
I'm not sure thats an easy task if we use this fairDispatcher instead of straight CasperPacketHandler. Or we have to have condition here. As I'm not sure we need fair dispatcher at all - I would prefer to enable it later if there is be a need.